lib/Ama.pm

```diff
@@ -5,6 +5,8 @@ use Ama::Model::Questions;
 use Ama::Model::Comments;
 use Ama::Model::Answers;
 use Ama::Model::Votes;
+use Ama::Model::Emailfeedback;
```

Comment:
I'd suggest the name feedback as opposed to emailfeedback.  The feature is for feedback, *how* you deliver that feedback should be irrelevant to the design.  That is, by choice, you should be able to change from email to posting a github issue.  But when it's called emailfeedback, that module is now misleading!